### ISSUE TITLE
Building the non-Docker version on Windows still requires Docker to be installed and open via default port

### ISSUE THREAD

**Issue by PolliSoft** [StackTraceYo/plex-bot#38]
When building the non-Docker version according to the instructions, you get a build error that Docker is not reachable via the Docker standard port.
The only way to build it is to install Docker and expose it via the default port.

**Comment by StackTraceYo** [comment]
@PolliSoft yeah i totally forgot that its in the pom. it still will build the jar for you itll just fail at that step. you can either comment out the docker plugin in the pom.xml or just ignore it. the jar is built before that step, look in the target directory and youll see it

**Comment by StackTraceYo** [comment]
@PolliSoft did you figure this out? i will make a ticket to fix this

**Comment by PolliSoft** [comment]
Yep, the jar file is built OK, thanks a lot!
